Jenkinsfile.nightlyadhoc

```diff
@@ -175,33 +176,50 @@ stage('Build Applications'){
     parallel stepsForParallelBuilds
 }
 
+def transitionIssueList
+def passedBuilds
+stage('JIRA'){
+    withEnv(['JIRA_SITE=cliqztix']){
+        def returnList = jira.transitionIssues()
+        passedBuilds = returnList[0]
+        transitionIssueList = returnList[1]
+    }
+}
+
 stage('Send email') {
     def projectName  = '$PROJECT_NAME'
+    def changesSinceLastSuccess = jira.getChangeString(passedBuilds)
     def bodytxt = """
         ${projectName} - Build # ${env.BUILD_NUMBER}
         Check console output at ${env.JOB_URL}${env.BUILD_NUMBER}/ to view the results.
         """
     params.each { prop, val ->
-        if(val){
-            def splitText = prop.toString().toLowerCase().split("_")
-            def brand = ""
-            def suffix = ""
-            if (splitText.size()==3){
-                if (splitText[2] == "x86"){
-                    suffix = "_x86"
-                }
-                brand = "${splitText[0]}-${splitText[1]}"
-            } else {
-                if (splitText[1] == "x86"){
-                    suffix = "_x86"
+        if(prop != "BRANCH"){
+            if(val){
+                def splitText = prop.toString().toLowerCase().split("_")
+                def brand = ""
+                def suffix = ""
+                if (splitText.size()==3){
+                    if (splitText[2] == "x86"){
+                        suffix = "_x86"
+                    }
+                    brand = "${splitText[0]}-${splitText[1]}"
+                } else {
+                    if (splitText[1] == "x86"){
+                        suffix = "_x86"
+                    }
+                    brand = "${splitText[0]}"
                 }
-                brand = "${splitText[0]}"
+                bodytxt = """${bodytxt}
+                ${brand}${suffix} apk Uploaded to: http://repository.cliqz.com.s3.amazonaws.com/dist/android/nightly/${params.BRANCH}/${brand}/${brand}_${env.BUILD_NUMBER}_${id}${suffix}.apk
```

Comment:
this url looks like a copy paste of ` utils.s3Copy` one. Maybe we should share them in some way?